fe/fe-core/src/main/java/com/starrocks/system/ComputeNode.java

```diff
@@ -89,7 +89,7 @@ public class ComputeNode implements IComputable, Writable, GsonPostProcessable {
     private AtomicBoolean isAlive;
 
     @SerializedName("isDecommissioned")
-    private final AtomicBoolean isDecommissioned;
+    private AtomicBoolean isDecommissioned;
```

Comment:
`isDecommissioned` is used for `alter system decommission backend`.
if you need `not alive` backend metric, you can use `total metric - alive metric` or add a new metric.